src/Migrations/Version20210812125029.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\InvoicingPlugin\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210812125029 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add payment_state to invoice';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE sylius_invoicing_plugin_invoice ADD payment_state VARCHAR(255) NOT NULL');
```

Comment:
`VARCHAR(255)` for such a short string gives me ♋ ( but i don't ask to change it - it can just create some performance and size related issues')